src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Queries/GetProvider/GetProviderQueryValidatorTests.cs

```diff
@@ -0,0 +1,44 @@
+using FluentValidation.Results;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetProvider;
+
+namespace SFA.DAS.CommitmentsV2.UnitTests.Application.Queries.GetProvider
+{
+    [TestFixture]
+    [Parallelizable]
+    public class GetProviderQueryValidatorTests
```

Comment:
This is fine as there is no state being set

Reply:
After discussion with Ian and Paul, this is not an issue. The parallelizable attribute on the class only makes the class run in parallel with other parallelizable classes. It doesn't make the elements within class run in parallel with each other, so this is not an issue.